### PR TITLE
Monitor: support PID configuration

### PR DESCRIPTION
**Pull request by ecourtois**

### CODE REVIEW

File: cogip/widgets/wizard.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+                padding: 6px;
+            }
+            QRadioButton#blueCamp:checked {
+                border-color: beige;
+                border-style: outset;
+            }
+            QRadioButton#blueCamp::indicator {
+                border-width: 0;
+            }
+            """)
```
Comment (gdoffe) [inline cogip/widgets/wizard.py:262]: Just a question, not more simple to put this in a seperate file ?

File: docs/usage/detector.md [hunk 2]
```diff
@@ -49,6 +49,10 @@ Options:
                                   env var: ROBOT_ID, COPILOT_ID
                                   default: 1; x>=1
 
+  -p, --uart-port PATH           Serial port connected to the Lidar
```
Comment (gdoffe) [inline docs/usage/detector.md:52]: 'Serial' word indent, missing one space

**Review by gdoffe — APPROVED** [review]
1 question and 1 typo. ;)
Goof job, I approve, just fix the typo and merge. ;)

File: cogip/widgets/wizard.py (hunk 1 above)
Comment (ecourtois) [reply cogip/widgets/wizard.py:262]: No that's the way to do it in Qt...

File: docs/usage/detector.md (hunk 2 above)
Comment (ecourtois) [reply docs/usage/detector.md]: Done

**Pull request merged**